### PR TITLE
fix: I don't want to talk about it

### PR DESCRIPTION
**Pull request by kaladivo**

### CODE REVIEW

File: apps/mobile/src/utils/lastTimeAppWasRunning.ts [hunk 1]
```diff
@@ -17,8 +16,7 @@ export function setLastTimeAppWasRunningToNow(): void {
 
 export function getLastTimeAppWasRunning(): UnixMilliseconds {
   return pipe(
-    localStorage.get(LAST_TIME_APP_WAS_RUNNING_KEY),
-    E.chain(safeParse(UnixMilliseconds)),
+    storage.getVerified(LAST_TIME_APP_WAS_RUNNING_KEY, UnixMilliseconds),
```
Comment (kaladivo) [inline apps/mobile/src/utils/lastTimeAppWasRunning.ts:19]: I really don't want to talk about it

File: apps/mobile/src/utils/notifications/checkForNewOffers.ts [hunk 2]
```diff
@@ -47,8 +46,7 @@ export function setLastNewOffersNotificationIssuedNow(): void {
 
 export function getLastNewOffersNotificationIssuedAt(): UnixMilliseconds {
   return pipe(
-    localStorage.get(LAST_NEW_OFFERS_NOTIFICATION_KEY),
-    E.chain(safeParse(UnixMilliseconds)),
+    storage.getVerified(LAST_NEW_OFFERS_NOTIFICATION_KEY, UnixMilliseconds),
```
Comment (kaladivo) [inline apps/mobile/src/utils/notifications/checkForNewOffers.ts:49]: Please, let's not talk about it

File: apps/mobile/src/utils/notifications/checkForNewOffers.ts [hunk 3]
```diff
@@ -71,6 +69,9 @@ async function displayNotification(t: TFunction): Promise<void> {
     },
     android: {
       channelId: await getDefaultChannel(),
+      pressAction: {
```
Comment (kaladivo) [inline apps/mobile/src/utils/notifications/checkForNewOffers.ts:72]: This is actually pretty interesting, this makes the press action work on every android device...

File: apps/mobile/src/utils/reportError.ts [hunk 4]
```diff
@@ -26,7 +26,11 @@ function reportError(lvl: LogLvl, message: string, ...args: any[]): void {
     crashlytics().log(toJsonWithRemovedSensitiveData({lvl, message, args}))
     crashlytics().recordError(new Error(removeSensitiveData(message)))
   }
-  getConsoleLvl(lvl)('‼️ there was an error reported. See hermes logs')
+  getConsoleLvl(lvl)(
+    '‼️ there was an error reported. See hermes logs',
+    message,
```
Comment (kaladivo) [inline apps/mobile/src/utils/reportError.ts:31]: At least print a message.

**Review by SamTremko — APPROVED** [review]

**Pull request merged**